storage-xray-udp/src/main/java/zipkin2/storage/xray_udp/UDPMessageEncoder.java

```diff
@@ -173,6 +176,12 @@ static void writeJson(Span span, Buffer buffer) throws IOException {
           case "aws.table_name":
             awsTableName = entry.getValue();
             continue;
+          case "aws.ec2.availability_zone":
```

Comment:
this seems like it would be better for initialization time vs based on tags. for example, the tracer isn't likely to change availability zones mid request.

Reply:
Hey, @adriancole! Thanks for the review, would it require some global change? E.g. `awsRegion` and `awsAccountId` might be good candidates for this too. 

I also tried to understand how this can be done, but I'm not sure what you mean by `initialization time`. AFAICS, `z.s.x.XRayUDPStorage` is a consumer that can only operate on a list of spans, and the information about the availability zones and the instance ids is only available on the instance that created the spans whereas running Zipkin doesn't have this information. Do you mean that this information can be transferred in span without using tags?

Reply:
sorry I was thinking this was the reporter not the storage.

I'm ok with this.. would be nice to hear if @devinsba is